src/java.base/share/classes/java/util/zip/ZipEntry.java

```diff
@@ -642,16 +647,19 @@ public byte[] getExtra() {
 
     /**
      * Sets the optional comment string for the entry.
-     *
-     * <p>ZIP entry comments have maximum length of 0xffff. If the length of the
-     * specified comment string is greater than 0xFFFF bytes after encoding, only
-     * the first 0xFFFF bytes are output to the ZIP file entry.
-     *
      * @param comment the comment string
-     *
+     * @throws IllegalArgumentException if the combined length
+     * of the specified entry comment, {@linkplain #getName() entry name},
```

Comment:
'entry name' is preceded by the definite article 'the' in `setExtra`, but not here. Would be good to be consistent.

Reply:
Done